src/connection.ts

```diff
@@ -1599,6 +1604,14 @@ export class Connection<S extends Schema = Schema> extends EventEmitter {
    * Module which manages process rules and approval processes
    */
   process = new Process(this);
+
+  public isLightningInstance(): boolean {
```

Comment:
let's make this private, it doesn't take URLs (in that case it would probably an util function) and after you create a connection it's guaranteed it doesn't use a lightning domain.